packages/odyssey-react/src/components/Heading/Heading.module.scss

```diff
@@ -12,34 +12,34 @@
 
 .root {
   margin-block-start: 0;
-  margin-block-end: $spacing-xs-em;
+  margin-block-end: var(--MarginBlockEnd);
   margin-inline: 0;
-  color: $text-heading;
+  color: var(--ColorHeading);
   font-weight: 600;
 }
 
 .level1 {
-  font-size: $size-title-1;
+  font-size: var(--SizeTitle1);
```

Comment:
Title is an interesting choice here since the component is no longer named that way. Are you trying to align with some figma token name here?

Now the current name system also has us naming these like so: `FontSize` rather than just `Size`

Reply:
Yes actually, we still use the title nomenclature in places like Infobox, Alert, and Banner, not to mention Figma. I can change this, I don't feel strongly.

Reply:
@arnoldsandoval-okta good reminders thanks - lets get these final places updated in a breaking change alongside 0.9.0 if we have time!